app/views/defence_requests/form_partials/_case.html.erb

```diff
@@ -1,42 +1,46 @@
 <h2><%= t('case_details') %></h2>
 <div class="case-details">
+  <%= f.text_field_input(:custody_number) %>
 
-  <div class="form-group">
-    <%= f.label :custody_number, label_text_for_form(attribute_name: 'custody_number'), class: "form-label-bold" %>
-    <%= f.text_field :custody_number, label: t('custody_number'), class: 'text-field' %>
-  </div>
-
-  <%= render('shared/date_time_form_part',
-             attribute: :time_of_arrest,
-             show_label: true,
-             dates: [Day::YESTERDAY, Day::TODAY],
-             initial_date_index: 1,
-             f: f) %>
+  <%= f.form_group(:time_of_arrest) do %>
+    <% render('shared/date_time_form_part',
```

Comment:
Here we combine the form builder and a partial, which then uses some parts of the builder. I'm just thinking, that it would be nice if we could just use the builder as all the components should be pre-defined and then we can just compose all forms from it. But maybe that's too complicated?